common/changes/@uifabric/dashboard/cardAccessability_2018-12-22-17-20.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
```

Comment:
>changes [](start = 3, length = 7)

is it possible to change the name of the file to have 'accessibility' instead of 'accessability'